tools/vitessce/main_macros.xml

```diff
@@ -1,11 +1,11 @@
 <macros>
-    <token name="@TOOL_VERSION@">1.0.4</token>
-    <token name="@VERSION_SUFFIX@">5</token>
+    <token name="@TOOL_VERSION@">3.5.1</token>
+    <token name="@VERSION_SUFFIX@">0</token>
     <token name="@PROFILE@">20.01</token>
```

Comment:
Should probably update this version too.

Reply:
What version should we bump the profile up to? version 22 or 23 (can't remember which) so it still works on the cancer instance for the time being?